pkg/analyzer/analyzer.go

```diff
@@ -63,16 +64,14 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	for _, f := range funcs {
 		for _, b := range f.Blocks {
 			for i := range b.Instrs {
-				// Check if instruction is call that returns a target type
-				targetValues := getTargetTypesValues(b, i, targetTypes)
-				if len(targetValues) == 0 {
+				// Check if instruction is call that returns a target pointer type
+				pointerTargetValues := getTargetTypesValues(b, i, targetTypes)
+				if len(pointerTargetValues) == 0 {
 					continue
 				}
 
-				// log.Printf("%s", f.Name())
-
 				// For each found target check if they are closed and deferred
-				for _, targetValue := range targetValues {
+				for _, targetValue := range pointerTargetValues {
```

Comment:
These are only sometimes pointers now, right? Should "pointer" still be added here? Am I missing something?